string object_name
---
bool success
string room_found
string object_found
---

